README.md

```diff
@@ -59,18 +59,19 @@ requires and provides, the permissions, and the additional module metadata.
-| ENV         |     folio     | Environment                                                                                                                                            |
-| KAFKA_HOST  |     kafka     | Kafka broker hostname                                                                                                                                  |
-| KAFKA_PORT  |     9092      | Kafka broker port                                                                                                                                      |
-| ENV         |     folio     | Logical name of the deployment, must be set if Kafka/Elasticsearch are shared for environments, `a-z (any case)`, `0-9`, `-`, `_` symbols only allowed |
-
+| Name                   | Default value   | Description                                                                                                                                            |
+|:-----------------------|:---------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------|
+| DB_HOST                |    postgres     | Postgres hostname                                                                                                                                      |
+| DB_PORT                |      5432       | Postgres port                                                                                                                                          |
+| DB_USERNAME            |   folio_admin   | Postgres username                                                                                                                                      |
+| DB_PASSWORD            |        -        | Postgres username password                                                                                                                             |
+| DB_DATABASE            |  okapi_modules  | Postgres database name                                                                                                                                 |
+| ENV                    |      folio      | Environment                                                                                                                                            |
+| KAFKA_HOST             |      kafka      | Kafka broker hostname                                                                                                                                  |
+| KAFKA_PORT             |      9092       | Kafka broker port                                                                                                                                      |
+| ENV                    |      folio      | Logical name of the deployment, must be set if Kafka/Elasticsearch are shared for environments, `a-z (any case)`, `0-9`, `-`, `_` symbols only allowed |
```

Comment:
**ENV** - This property is added twice in this file. Please remove from one place

Reply:
done